generators/boilerplate/templates/index.js

```diff
@@ -1,3 +1,4 @@
-'use strict';
+<% if (babel) { %>export default {};
```

Comment:
Is this working with Node.js? Even though es6 have modules, that doesn't mean node loading system support them yet.